src/group_theory/group_action/defs.lean

```diff
@@ -778,3 +759,45 @@ See note [reducible non-instances]. -/
 @[reducible]
 def add_action.of_End_hom [add_monoid M] (f : M →+ additive (function.End α)) : add_action M α :=
 add_action.comp_hom α f
+
+/-! ### `additive`, `multiplicative` -/
+
+section
+
+open additive multiplicative
+
+instance additive.has_vadd [has_smul α β] : has_vadd (additive α) β := ⟨λ a, (•) (to_mul a)⟩
+instance multiplicative.has_smul [has_vadd α β] : has_smul (multiplicative α) β :=
+⟨λ a, (+ᵥ) (to_mul a)⟩
```

Comment:
`(additive.)to_mul` is the wrong name here. I recommend possibly not opening the namespace, it makes getting the names right more confusing
```suggestion
⟨λ a, (+ᵥ) (to_add a)⟩
```

Reply:
Now you understand https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/multiplicative.2C.20additive !